### ISSUE TITLE
Accessibility review - round II

### ISSUE THREAD

**Issue by samikeijonen** [easydigitaldownloads/easy-digital-downloads#4842]
In ticket #4031 we handled some of the accessibility issues. I can review second round for `a11y`. What would you like to check next?

**Comment by pippinsplugins** [comment]
Yes please! Could we focus on the view order details screen and the settings screens?

**Comment by samikeijonen** [comment]
Yes we could. I'll take a look at these next. I can't promise any timetable though.
